argusAI/event_similarity_processor.py

```diff
@@ -0,0 +1,274 @@
+import os
+from collections import namedtuple
+from pathlib import Path
+import chromadb
+from threading import Thread, Event
+import time
+import logging
+import datetime
+import uuid
+
+from chromadb.utils.embedding_functions.onnx_mini_lm_l6_v2 import ONNXMiniLM_L6_V2
+from chromadb.config import Settings
+
+import itertools
+from argus.backend.models.argus_ai import ErrorEventEmbeddings, CriticalEventEmbeddings
+from argus.backend.plugins.sct.testrun import SCTTestRun
+from argusAI.utils.event_message_sanitizer import MessageSanitizer
+from argusAI.utils.scylla_connection import ScyllaConnection
+
+SLEEP_INTERVAL = 300  # 5 minutes
+SIMILARITY_THRESHOLD = 0.90
+MAX_SIMILARS = 20
+DAYS_BACK = int(os.getenv('DAYS_BACK', 120))
+
+logging.basicConfig(
```

Comment:
You could use the logging module we have on the backend so that message format would be the same between those two.